### PR TITLE
Remove unnecessary javascript code and replace with hugo template code

### PR DESCRIPTION
**Pull request by susnux**
Supersede #87

### CODE REVIEW

**Comment by zjedi** [comment]
Does this fix the flickering of down-arrows?

File: assets/js/index.js [hunk 1]
```diff
@@ -48,25 +48,6 @@ var $sitehead = $("#site-head");
...
-      var t = $(this).text();
-      var index = $(this).parents(".post-holder").index();
-      $fnav.append(
-        "<a class='fn-item' item_index='" + index + "'>" + t + "</a>"
-      );
-      $(".fn-item").click(function () {
-        var i = $(this).attr("item_index");
-        var s = $(".post[item_index='" + i + "']");
-
-        $("html, body").animate(
```
Comment (zjedi) [inline assets/js/index.js:61]: Tthis removes the smooth animation of scrolling to content, which I believe is one of characteristic features of this theme and should be preserved.

I think 400ms scrolltime is too much, 150ms fits me better, but I'd keep it animated.  There are also some side-issues with this feature I am aware of, but I'd rather fix them.

Comment (susnux) [reply assets/js/index.js:61]: ok will look into this

**Comment by zjedi** [comment]
Feel free to reopen when animation is brought back :)

**Pull request closed without merging**